user.go

```diff
@@ -216,6 +347,57 @@ func (s *Server) handlerUserLogout(w http.ResponseWriter, r *http.Request) {
 	http.Redirect(w, r, "/", http.StatusFound)
 }
 
+func (s *Server) AddAuthMethodToUser(auth *common.AuthMethod, user *common.User) (*common.User, error) {
```

Comment:
These two functions could be relocated into the dataconnector, they do not need the server dependency

Reply:
nvm, lets leave it in the server for now, it would require changing the interface etc.